.gitignore

```diff
@@ -40,3 +40,4 @@ bin/
 
 ### Mac OS ###
 .DS_Store
+/.idea/
```

Comment:
Add this above, in IntelliJ IDEA section, right now it's in Mac OS section. Also remove the other .idea entries, not needed anymore.